src/app-modules/docker-compose

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -x
+
+docker_compose_cmd=docker-compose
+docker_cmd=docker
+
+app_load() {
+    local url="$1"
+    local input_file="$2"
+
+    $docker_cmd image load < "$input_file"
+}
+
+app_save() {
+    local url="$1"
+    local output_file="$2"
+
+    $docker_cmd image save "$url" -o "$output_file"
+}
+
+app_rollout() {
+    local manifests_dir="$1"
+
+    (cd "$manifests_dir" && $docker_compose_cmd up -d > compose.log 2>&1 3>&1) &
+}
+
+app_stop() {
+    local manifests_dir="$1"
```

Comment:
Also here, according to the (amended) docs this should be `$2`.

Reply:
fixed. I also went through remaing API calls, and make them accept always application name as the first argument.